core/jazz_slack-event-handler/index.js

```diff
@@ -0,0 +1,303 @@
+      .catch(error => {
+        return reject(error);
+      });
+  });
+}
+
+function getSvcPayload(method, payload, apiEndpoint, authToken) {
+  let svcPayload = {
+    headers: {
+      'content-type': "application/json",
+      'authorization': authToken
+    },
+    rejectUnauthorized: false
+  };
+
+  svcPayload.uri = apiEndpoint;
+  svcPayload.method = method;
+  if (payload) {
+    svcPayload.json = payload;
+  }
+  logger.debug("payload :" + JSON.stringify(svcPayload));
+  return svcPayload;
+}
+
+function processRequest(svcPayload) {
+  return new Promise((resolve, reject) => {
+    request(svcPayload, function (error, response, body) {
+      if (response.statusCode === 200 && body) {
+        return resolve(body);
+      } else {
+        logger.error("Error processing request: " + JSON.stringify(response));
+        return reject(error);
+      }
+    });
+  });
+}
+
+function getServiceDetails(eventPayload, configData, authToken) {
+  return new Promise((resolve, reject) => {
+    let service_id = eventPayload.SERVICE_ID.S;
+    let apiEndpoint = configData.SERVICE_API_URL + configData.SERVICE_API_RESOURCE + "/" + service_id;
+    let svcPayload = exportable.getSvcPayload("GET", null, apiEndpoint, authToken);
+
+    exportable.processRequest(svcPayload)
+      .then(result => { return resolve(result); })
+      .catch(err => {
+        logger.error("getServiceDetails failed: " + JSON.stringify(err));
+        let error = exportable.handleError(failureCodes.PR_ERROR_1.code, failureCodes.PR_ERROR_1.message);
+        return reject(error);
+      });
+  });
+}
+
+function notifySlackChannel(result, payload, configData, authToken) {
+  return new Promise((resolve, reject) => {
+    let output = JSON.parse(result);
+    if (!output.data) {
+      logger.error("Service details not found in service catalog");
+      let error = exportable.handleError(failureCodes.PR_ERROR_1.code, failureCodes.PR_ERROR_1.message);
+      return reject(error);
+    }
+    let serviceDetails = output.data;
+    logger.debug("service details : " + JSON.stringify(serviceDetails));
+    if (!serviceDetails.slack_channel) {
+      logger.error("Slack channel not found");
+      let error = exportable.handleError(failureCodes.PR_ERROR_5.code, failureCodes.PR_ERROR_5.message);
+      return reject(error);
+    }
+
+    let attachments = [];
+    let notification = utils.getNotificationMessage(serviceDetails, payload, configData);
+    logger.debug("notification details : " + JSON.stringify(notification));
+    if (notification) {
+      attachments.push(utils.formatSlackTemplate(
+        notification.pretext,
+        notification.text,
+        notification.color
+      ));
+    }
+
+
+    let slackNotificationSvcPayload = {
+      "method": "POST",
+      "uri": configData.SLACK_BASIC_NOTIFICATION_URL + "?token=" + configData.SLACK_TOKEN + "&channel=" + serviceDetails.slack_channel + "&username=" + configData.SLACK_NOTIFIER_USER_NAME,
```

Comment:
can you backticks to help with formatting which will ease readability of code? Here is an example: https://github.com/tmobile/jazz/blob/v1.7.1/core/jazz_usermanagement/components/config.js#L48